LICENSE.md

```diff
@@ -270,3 +270,16 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ### contextMenu [2.1.1](https://github.com/swisnl/jQuery-contextMenu)
 MIT licensed
+
+--------------------------------------------------------------------------------
+
+### FileSaver [1.1.20160328](https://github.com/eligrey/FileSaver.js/)
+### Blob [2014-07-24](http://purl.eligrey.com/github/Blob.js/blob/master/Blob.js)
```

Comment:
It's the same license since it's made by the same guy, which is why I have the two together. How would you like me to break it up?